tools/README.md

```diff
@@ -0,0 +1,110 @@
+# Istio Load Testing User Guide
+This guide provides step-by-step instructions for using the `setup_perf_cluster.sh` load testing script.
+The script deploys a GKE cluster, an Istio service mesh, a GCE VM and runs [Fortio](https://github.com/istio/fortio/)
+on the VM and within the mesh. Fortio is used to perform load testing.
+
+### Clone Istio
+```
+$ git clone https://github.com/istio/istio.git && cd istio
+```
+
+### Prepare the Istio Deployment Manifest and Istio Client
+__Option A:__ Build the deployment manifest and `istioctl` binary:
+```
+$ ./install/updateVersion.sh
+```
+Follow the steps in the [Developer Guide](https://github.com/istio/istio/blob/master/DEV-GUIDE.md) to build the `istioctl` binary.
+Make the kubectl binary executable.
+```
+$ chmod +x ./istioctl
+```
+
+Move the binary in to your PATH.
+```
+$ mv ./istioctl /usr/local/bin/istioctl
+```
+
+__Option B:__ Follow the [quickstart guide](https://istio.io/docs/setup/kubernetes/quick-start.html) to install the
+manifests and `istioctl` binary. Make sure `istioctl` in your path is the one matching the downloaded release.
+For instance, in `~/tmp/istio-0.4.0/` run:
+```
+$ ln -s $GOPATH/src/istio.io/istio/tools
```

Comment:
:+1:

nice ! did I mention that somewhere or you discovered it ?

Reply:
You mentioned it in the script. I moved some of your notes out of the script into the readme. Teamwork :-)